pandas/core/indexes/base.py

```diff
@@ -5023,8 +5023,9 @@ def array(self) -> ExtensionArray:
             from pandas.core.arrays.numpy_ import NumpyExtensionArray
 
             array = NumpyExtensionArray(array)
-        array = array.view()
-        array._readonly = True
+        # TODO decide on read-only https://github.com/pandas-dev/pandas/issues/63099
+        # array = array.view()
+        # array._readonly = True
```

Comment:
is the idea to leave this commented-out indefinitely?  or will be cleaned up before merging?